src/org/labkey/test/components/react/DropdownButtonGroup.java

```diff
@@ -0,0 +1,297 @@
+/*
+ * Copyright (c) 2018-2019 LabKey Corporation. All rights reserved. No portion of this work may be reproduced in
+ * any form or by any electronic or mechanical means without written permission from LabKey Corporation.
+ */
+package org.labkey.test.components.react;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebDriverWrapperImpl;
+import org.labkey.test.components.WebDriverComponent;
+import org.labkey.test.util.LogMethod;
+import org.labkey.test.util.LoggedParam;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+import static org.labkey.test.WebDriverWrapper.sleep;
+
+
+public class DropdownButtonGroup extends WebDriverComponent<DropdownButtonGroup.ElementCache>
+{
+    final WebElement _el;
+    final WebDriver _driver;
+    private Integer _expandRetries = 1;
```

Comment:
I don't think it is ever accessed.

Reply:
Are you requesting its removal?

Reply:
Having an used private variable is not a big deal, but in this case there is a public interface that sets it (see withExpandRetries(Integer expandRetries) below) and never uses it. Because of that it should be cleaned up.

This is where a linter would help because it can call out unused variables.